profiles/base.yaml

```diff
@@ -2219,3 +2219,21 @@ skill_recorder_check_exp_mods: false
 # The polling interval reduces spamming commands while we wait
 # for your dead body to be resurrected so that we can run `info` again.
 drinfomon_passive_delay: 30
+
+# For Dragonrealms on Lich5 only
+# A change in behavior from Lich4 to Lich5 caused certain ;go2 map wayto movements to fail.
+# This is because of the way the map stringprocs are structured, and the method used to evaluate
+# certain map moves, which involved an arguably dubious overloading of FalseClass on Lich4.
+# The transition to Lich5 makes it necessary to fix some stringprocs as an interim solution, at
+# least until the map can be divested from current control and handled appropriately. This section
+# FIXES via stringproc known map movement failures as a result of the transition to Lich5. A user
+# with the appropriate skill can overload these in their own yaml(s).
+base_wayto_overrides:
+  aksetis_mount_up:
+    start_room: 27558
```

Comment:
Should this be 27556 or are we defaulting to letting a base movement do the first climb up?